code/builders/builder-manager/src/index.ts

```diff
@@ -317,3 +350,6 @@ export const build: ManagerBuilder['build'] = async (options) => {
 
 export const corePresets: ManagerBuilder['corePresets'] = [];
 export const overridePresets: ManagerBuilder['overridePresets'] = [];
+function sanatize(arg0: string | undefined) {
```

Comment:
Minor: It's `sanitize`.

More importantly: I didn't see this function called anywhere and it's not exported?